crates/sc-consensus-subspace/src/lib.rs

```diff
@@ -815,7 +818,7 @@ pub struct SubspaceBlockImport<Block: BlockT, Client, I, CAW, CIDP> {
     subspace_link: SubspaceLink<Block>,
     can_author_with: CAW,
     create_inherent_data_providers: CIDP,
-    maybe_block_import_throttling_sender: Option<mpsc::Sender<()>>,
+    executor_enabled: bool,
```

Comment:
Let's remove this parameter, it doesn't matter whether executor is enabled or not here. `block_import_throttling_sender` is also a strange name, but we can probably come up with a better one later.

Reply:
We're not enforcing anything, this is just unnecessary complexity. If you drop the sender, there will effectively be no throttling. This is also the reason check for messages received in `.is_none` I commented 500 lines below is unnecessary too.

Reply:
Okay, relying on the sender drop makes it simpler yet less explicit. Updated.

Reply:
And any idea on the naming?

Reply:
Some kind of acklowledgement I think.

Reply:
Then `acknowledge_block_import_sender` or `block_import_acknowledgement_sender`? Feel free to suggest one, no strong preference on this naming.

Reply:
`block_import_acknowledgement_sender` seems to sound better